preppal-fe/src/pages/new-recipe.tsx

```diff
@@ -0,0 +1,96 @@
+                        </Form.Group>
+                        <MdCancel size={30}/>
+                    </Stack>
+
+                    <div style={{ paddingBottom: '24px' }}>
+                        <Button className="mx-auto" variant="primary" title="Add Ingredient" size="sm">
+                            Add Ingredient
+                        </Button>
+                    </div>
+
+                    Directions
+                    <Stack direction="horizontal" gap={2}>
+                        <Form.Group controlId="recipeStep" style={{ paddingBottom: '8px', width: '90%' }}>
+                            <Form.Control type="step" />
+                        </Form.Group>
+                        <MdCancel size={30}/>
+                    </Stack>
+                    <div style={{ paddingBottom: '24px' }}>
+                        <Button className="mx-auto" variant="primary" title="Add Step" size="sm">
+                            Add Step
+                        </Button>
+                    </div>
+
+                    <Form.Group controlId="recipeServingSize" style={{ paddingBottom: '24px' }}>
+                        Serving Size
+                        <Form.Control type="servingSize" />
+                    </Form.Group>
+
+                    Prep Time
+                    <Form.Group controlId="recipePrepTime" style={{ paddingBottom: '24px' }}>
+                        <Stack direction="horizontal">
+                            <Form.Control type="prepTime"/>
+                            <Form.Select style={{ width: '50%'}}>
+                                <option value={1}>mins</option>
+                                <option value={2}>hours</option>
+                            </Form.Select>
+                        </Stack>
+                    </Form.Group>
+
+                    <Form.Group controlId="recipeCookeTime" style={{ paddingBottom: '24px' }}>
+                        Cook Time
+                        <Stack direction="horizontal">
+                            <Form.Control type="cookTime"/>
+                            <Form.Select style={{ width: '50%'}}>
+                                <option value={1}>mins</option>
+                                <option value={2}>hours</option>
+                            </Form.Select>
+                        </Stack>
+                    </Form.Group>
+
+                    <div className="float-end" style={{ paddingBottom: '16px' }}>
+                        <Button className="mx-auto" variant="tertiary" title="Cancel" size="lg">
+                            Cancel
+                        </Button>
+                        <Button className="mx-auto" variant="primary" type="submit" title="Submit" size="lg">
+                            Submit
+                        </Button>
+                    </div>
+                </Form>
+            </div>
+        </Container>
+    );
+};
+
+export default LoginPage;
```

Comment:
Change the export default to NewRecipe.
Also add new routing to app.ts
I think we just add a link for this page to the navbar and do cleanup later.